### PR TITLE
CY-1109 Add support for missing relationships

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

File: openstacksdk_plugin/resources/compute/server.py [hunk 1]
```diff
@@ -953,6 +946,40 @@ def _get_user_password(openstack_resource):
...
+    This method will help to remove connection between port and security group
+    Because when we attach security group to a server that has multiple
+    ports connected to it, all the ports automatically are going to connect
+    to the security group
+    :param dict client_config: Openstack configuration required to connect
+    to API
+    :param dict server_payload: Server payload configuration from openstack
+    :param str security_group_id: Security group ID
+    """
+    if server_payload:
```
Comment (EarthmanT) [inline openstacksdk_plugin/resources/compute/server.py]: Why not:
```python
if not server_payload:
  return
```

File: plugin.yaml [hunk 2]
```diff
@@ -1282,6 +1282,43 @@ relationships:
   cloudify.relationships.openstack.server_connected_to_port:
     derived_from: cloudify.relationships.connected_to
 
+  cloudify.relationships.openstack.server_connected_to_floating_ip:
```
Comment (EarthmanT) [inline plugin.yaml:1294]: Should we add a message here that his is deprecated in the API?

**Review by EarthmanT — APPROVED** [review]
Before merge - respond to comments.

File: plugin.yaml (hunk 2 above)
Comment (mabuaisha) [reply plugin.yaml:1294]: @EarthmanT I found the issue for this, it is solved now, the API is deprecated but we still can use it , the API is working it seems there is a little issue with calling the API

File: openstacksdk_plugin/resources/compute/server.py (hunk 1 above)
Comment (mabuaisha) [reply openstacksdk_plugin/resources/compute/server.py]: Yeah We can do it on this way, will update it

**Pull request merged**